fastlane/Fastfile

```diff
@@ -316,8 +316,8 @@ end
 
 platform :ios do
   before_all do
-    if ENV['CIRCLECI'] == 'true'
-      setup_circle_ci
+    if ENV['CI'] == 'true'
+      setup_ci
```

Comment:
What does this do?

Reply:
Chiming in briefly to address comments on changes I did @toninis 
[setup_ci](https://docs.fastlane.tools/actions/setup_ci/) seems to act almost exactly like [setup_circle_ci](https://docs.fastlane.tools/actions/setup_circle_ci/), it creates a temporary keychain and little else. That's what the [Fastlane docs for Github Actions](https://docs.fastlane.tools/best-practices/continuous-integration/github/) suggest using